src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchEngine.cs

```diff
@@ -54,7 +54,7 @@ internal partial class FindReferencesSearchEngine
             _dependencyGraph = solution.GetProjectDependencyGraph();
             _options = options;
 
-            _progressTracker = new StreamingProgressTracker(progress.ReportProgressAsync);
+            _progressTracker = progress.ProgressTracker;
```

Comment:
here's an actual client that pushes progress information into the find-refs-window.  They don't need to update and of their progress code (since they were already pushign into an IStreamingProgressTracker).  Instead, they can just delegate to the window itself.